scripts/spack/configs/blueos_3_ppc64le_ib_p9/spack.yaml

```diff
@@ -0,0 +1,351 @@
+      externals:
+      - spec: groff@1.22.2
+        prefix: /usr
+    libtool:
+      buildable: false
+      externals:
+      - spec: libtool@2.4.2
+        prefix: /usr
+    libunwind:
+      buildable: false
+      externals:
+      - spec: libunwind@8.0.1
+        prefix: /usr
+    ncurses:
+      buildable: false
+      externals:
+      - spec: ncurses@5.9
+        prefix: /usr
+    m4:
+      buildable: false
+      externals:
+      - spec: m4@1.4.16
+        prefix: /usr
+    perl:
+      buildable: false
+      externals:
+      - spec: perl@5.16.3
+        prefix: /usr
+    pkg-config:
+      buildable: false
+      externals:
+      - spec: pkg-config@0.27.1
+        prefix: /usr
+    readline:
+      buildable: false
+      externals:
+      - spec: readline@6.2
+        prefix: /usr
+    tar:
+      externals:
+      - spec: tar@1.26
+        prefix: /usr
+      buildable: false
+
+    # External dependencies for SCR
+    lsf:
+      buildable: False
+      externals:
+      - spec: lsf@10.1
+        prefix: /opt/ibm/spectrumcomputing/lsf/10.1
+    slurm:
+      buildable: false
+      externals:
+      - spec: slurm@20
+        prefix: /usr
+    libyogrt:
+      buildable: false
+      externals:
+      - spec: libyogrt@1.0.0 scheduler=lsf
+        prefix: /usr
+      - spec: libyogrt@1.0.0 scheduler=slurm
+        prefix: /usr
+    pdsh:
+      buildable: false
+      externals:
+      - spec: pdsh@2.33
+        prefix: /usr
+
+    # Globally lock version of third party libraries
+    conduit:
+      require: "@0.8.6~shared~test~examples~utilities"
+    hdf5:
+      variants: ~shared~mpi
+    hypre:
+      version: [2.24.0]
+    mfem:
+      require: "@4.5"
+    raja:
+      variants: ~shared~examples~exercises
+    umpire:
+      variants: ~shared~examples
+
+    # Globally lock in versions of Devtools
+    cmake:
+      version: [3.21.1]
+      buildable: false
+      externals:
+      - spec: cmake@3.21.1
+        prefix: /usr/tce/packages/cmake/cmake-3.21.1
+    cppcheck:
+      version: [2.9]
+      buildable: false
+      externals:
+      - spec: cppcheck@2.9
+        prefix: /collab/usr/gapps/axom/devtools/blueos_3_ppc64le_ib_p9/latest/cppcheck-2.9
+    doxygen:
+      version: [1.8.14]
+      buildable: false
+      externals:
+      - spec: doxygen@1.8.14
+        prefix: /collab/usr/gapps/axom/devtools/blueos_3_ppc64le_ib_p9/latest/doxygen-1.8.14
+    graphviz:
+      version: [7.1.0]
+      buildable: false
+      externals:
+      - spec: graphviz@7.1.0
+        prefix: /collab/usr/gapps/axom/devtools/blueos_3_ppc64le_ib_p9/latest/graphviz-7.1.0
+    llvm:
+      version: [10.0.0]
+      buildable: false
+      externals:
+      - spec: llvm@10.0.0+clang
+        prefix: /usr/tce/packages/clang/clang-10.0.0
+    python:
+      version: [3.10.10]
+      buildable: false
+      externals:
+      - spec: python@3.10.10
+        prefix: /collab/usr/gapps/axom/devtools/blueos_3_ppc64le_ib_p9/latest/python-3.10.10
```

Comment:
Weird -- looks like we now need to bake in the path to our devtools

Reply:
while latest is pointing to a different path, the prefixes looks practically identical as they did before

https://github.com/LLNL/axom/blob/b27dd885e1a1eddd122ca6f00d2b0d6243ca8832/scripts/spack/configs/blueos_3_ppc64le_ib_p9/packages.yaml#L241-L246